dronegen/common.go

```diff
@@ -0,0 +1,100 @@
+package main
+
+import "fmt"
+
+var (
+	triggerPullRequest = trigger{
+		Event: triggerRef{Include: []string{"pull_request"}},
+		Repo:  triggerRef{Include: []string{"gravitational/*"}},
+	}
+	triggerPush = trigger{
+		Event:  triggerRef{Include: []string{"push"}, Exclude: []string{"pull_request"}},
```

Comment:
nit: can you elaborate why this configures both includes and excludes? IIUC, for drone a single configuration suffices with excludes taking precedence. If it's an optimization, it's extremely negligible.

Reply:
I have a feeling that you're right and this statement is actually somewhat redundant in the current Drone config, but I was trying to avoid changing the YAML file too much with this PR - i.e. I just want `dronegen` to produce more or less exactly the same thing as we currently have to make migration easier. We can make improvements to tidy up and remove sections in much simpler PRs in future.